### DISCUSSION TITLE
ide-php syntax-error in Json-Mapper

### DISCUSSION THREAD

**Discussion by azGiEAhAtu** [atom/atom#23453] [Q&A]
Atom 1.58.0 x64 + ide-php (+ php analyzer)
Error at activating the ide:
PHP Fatal error: Array and string offset access syntax with curly braces is no longer supported in C:\Users\maxib.atom\packages\ide-php\vendor\netresearch\jsonmapper\src\JsonMapper.php on line 300 PHP Fatal error: Array and string offset access syntax with curly braces is no longer supported in C:\Users\maxib.atom\packages\ide-php\vendor\netresearch\jsonmapper\src\JsonMapper.php on line 300 DEBUG Restarted process exited 255 DEBUG Restarted process exited 255
Hints: PHP installed with XAMPP, 8.0.9, www-root ist not default, but works proper.
Has anyone an idea?

**Comment by icecream17** [comment]
error is probably this: https://stackoverflow.com/questions/59158548/array-and-string-offset-access-syntax-with-curly-braces-is-deprecated
somehow comes from this package: https://github.com/atom/ide-php
